backend/src/test/java/de/roman/toolio/service/InquiryPartServiceTest.java

```diff
@@ -0,0 +1,162 @@
+package de.roman.toolio.service;
+
+import de.roman.toolio.db.InquiryPartDb;
+import de.roman.toolio.model.InquiryPart;
+import de.roman.toolio.model.UuidGenerator;
+import de.roman.toolio.security.AppUser;
+import de.roman.toolio.security.AppUserDb;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.*;
+
+class InquiryPartServiceTest {
+    private final InquiryPartDb inquiryPartDb = mock(InquiryPartDb.class);
+    private final AppUserDb appUserDb = mock(AppUserDb.class);
+    private final UuidGenerator uuidGenerator = mock(UuidGenerator.class);
```

Comment:
I wonder why you generate uuids yourself?

Reply:
This is due to the fact, that I want to add the uuid of the added part to the posting user as well. If I would let mongoDb take care of the ID generation, I would not know the exact ID of the part and therefore cannot write it to the posting User. This information however, is required, since after login I want the user to have the possibility to see his posted Inquiries.

Reply:
Okay, this works, but after saving a new part to the db you could simply get the id from the returned object, I think.